### PR TITLE
Feat pagination

### PR DESCRIPTION
**Pull request by adrean-hipe**
### OVERVIEW

- added pagination for home, profile, and search controllers
- removed unused index method in PostController
- added pagination links to the pages

### SNIPPETS

home pagination
image
profile paginationi - 2nd page
image
search paginnation
image

### CODE REVIEW

File: laravel-app/app/Http/Controllers/ProfileController.php [hunk 1]
```diff
@@ -64,23 +64,25 @@ public function show(int $userId): RedirectResponse|View
...
-        });
-
         /** @var User $user */
         $imagePath = $this->getImagePath($user);
         $authUser = auth()->user();
         return view('profile.show', [
             'authUser' => $authUser,
             'user' => $user,
-            'likesCount' => $likesCount,
             'imagePath' => $imagePath,
```
Comment (bpocrafael) [inline laravel-app/app/Http/Controllers/ProfileController.php:84]: use model for the path

**Review by bpocrafael — CHANGES_REQUESTED** [review]

**Review by bpocrafael — APPROVED** [review]

File: laravel-app/resources/views/profile/edit.blade.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <div class="container text-center">
         <div class="row justify-content-center">
             <div class="col-md-4 text-center">
-                <img id="profileImage" class="img-fluid w-50" src="{{ asset($imagePath) }}" alt="Microblog Logo">
+                <img id="profileImage" class="img-fluid w-50" src="{{ asset('storage/'. $user->image_path) }}" alt="Microblog Logo">
```
Comment (bpocrafael) [inline laravel-app/resources/views/profile/edit.blade.php:8]: could you add the 'storage/' in the attribute also

File: laravel-app/resources/views/profile/show.blade.php [hunk 3]
```diff
@@ -8,7 +8,7 @@
 	<div class="p-3">
 		<div class="row justify-content-center">
 			<div class="col-md-4 text-center">
-				<img class="img-fluid w-50" src="{{asset($imagePath) }}" alt="Microblog Logo">
+				<img class="img-fluid w-50" src="{{ asset('storage/'. $user->image_path) }}" alt="Microblog Logo">
```
Comment (bpocrafael) [inline laravel-app/resources/views/profile/show.blade.php:11]: same with this. so that you would not to add the 'storage/'

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

File: laravel-app/app/Http/Controllers/ProfileController.php (hunk 1 above)
Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/ProfileController.php]: tranferred to model as attribute sir, thank you ❤️

File: laravel-app/resources/views/profile/edit.blade.php (hunk 2 above)
Comment (adrean-hipe) [reply laravel-app/resources/views/profile/edit.blade.php]: added storage to file path in attribute sir, thank you ❤️

File: laravel-app/resources/views/profile/show.blade.php (hunk 3 above)
Comment (adrean-hipe) [reply laravel-app/resources/views/profile/show.blade.php]: added storage to file path in attribute sir, thank you ❤️

**Review by bpocrafael — APPROVED** [review]

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**